src/msix/pack/AppxBlockMapWriter.cpp

```diff
@@ -53,8 +53,14 @@ namespace MSIX {
     }
 
     // <Block Size="2948" Hash="ORIk+3QF9mSpuOq51oT3Xqn0Gy0vcGbnBRn5lBg5irM="/>
-    void BlockMapWriter::AddBlock(const std::vector<std::uint8_t>& hash, ULONG size, bool isCompressed)
+    void BlockMapWriter::AddBlock(const std::vector<std::uint8_t>& block, ULONG size, bool isCompressed)
     {
+        // hash block
+        std::vector<std::uint8_t> hash;
+        ThrowErrorIfNot(MSIX::Error::SignatureInvalid, 

```

Comment:
>SignatureInvalid [](start = 37, length = 16)

Not the correct error to use; might need a new one.  Or maybe a generic "invalid data" error here.